app/jinja_filters.py

```diff
@@ -660,7 +669,7 @@ def map_list_collector_config(
     rows: list[dict[str, list] | SummaryRow] = []
 
     for index, list_item in enumerate(list_items, start=1):
-        item_name = list_item.get("item_title")
+        item_name = str(list_item.get("item_title", ""))
```

Comment:
It was to do with the type hints and casting type, I didn't want to cast `str` on `None` since the return value would be `"None"` which could be potentially end up in aria label but if this cannot be ever empty (since here we are always in the context of a given list item) it doesn't matter so I'll revert it.